### PR TITLE
Add chapter 1 questions

### PR DESCRIPTION
**Pull request by pbmasigla**

### CODE REVIEW

File: src/lib/questions/cracking-the-code.js [hunk 1]
```diff
@@ -1,14 +0,0 @@
-module.exports = [
-	{
-		question: "Who ate the banana?",
```
Comment (krissalvador27) [inline src/lib/questions/cracking-the-code.js]: 🍌

**Comment by pbmasigla** [comment]
@krissalvador27 do you think the question structure will be okay? I'm worried that it might be a large, inflexible structure

File: src/lib/questions/cracking-the-code/1-arrays-and-strings/1.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+			class: "text-question",
+			content: "What is the running time of this code?"
+		},
+		{
+			tag: "pre",
+			class: "code-question",
+			content: (
+				`
+public String makeSentence(String[] words) {
+	StringBuffer sentence = new StringBuffer();]
```
Comment (krissalvador27) [inline src/lib/questions/cracking-the-code/1-arrays-and-strings/1.js]: extra ] on end?

**Comment by krissalvador27** [comment]
theres some grammar on the questions that I wont be anal about .. I'll prob fix it up once I figure out a nice UI for it and update the styles

🐙

**Comment by pbmasigla** [comment]
@krissalvador27 the pdf that I'm copying from seems to be missing letters every now and then when I do a straight copy and paste :(

**Comment by pbmasigla** [comment]
@krissalvador27 added all questions. still need to add answers but that can be for later

**Comment by krissalvador27** [comment]
ill do the answers

**Pull request merged**